build/commands/lib/util.js

```diff
@@ -771,24 +812,6 @@ const util = {
     return applyPatches()
   },
 
-  buildArgsToString: (buildArgs) => {
-    let args = ''
-    for (let arg in buildArgs) {
-      let val = buildArgs[arg]
-      if (val !== null) {
-        if (typeof val === 'string') {
-          val = '"' + val + '"'
-        } else {
-          val = JSON.stringify(val)
-        }
-      }
-      args += val ? arg + '=' + val + ' ' : arg + ' '
-    }
-    // "D:\" should be escaped as \"D:\\\\\".
```

Comment:
Is it okay to drop it?
And
```
if (typeof val === 'string') {
   val = '"' + val + '"'`
```
above?

Reply:
yes, I've tested all these cases explicitly. The escaping we got from `JSON.stringify` covers everything we need to pass args via `.gni` file. The additional escaping was required only when we were using the command line.

this should just be `JSON.stringify(val)` as it does add quotes for `string` type automatically:
```
if (typeof val === 'string') {
   val = '"' + val + '"'`
```